src/olympia/amo/middleware.py

```diff
@@ -60,15 +60,16 @@ def process_request(self, request):
         ):
             return redirect_type(request.path.replace('/mobile', '/android'))
 
-        if 'lang' in request.GET and not re.match(
+        if ('lang' in request.GET or 'app' in request.GET) and not re.match(
```

Comment:
Do we really need / want this change ? Is it just for parity with how we deal with `?lang` ?

Reply:
I added this change when I found some existing tests for `lang`. Do we need it? No. I thought it would be slightly better to have the same behavior for both `lang` and `app` query params.